### DISCUSSION TITLE
Can't import the NFT after solving a challenge

### DISCUSSION THREAD

**Discussion by Aishwaria8786** [Cyfrin/foundry-full-course-cu#1532] [Q&A] [answered]
I solved the challenge of lesson 4 but I am having a problem in importing the NFT. I provided the correct address and token ID but still can't import it (Metamask shows the message below). What should I do to solve it?
![WHY](https://github.com/Cyfrin/foundry-full-course-f23/assets/68544977/d750507e-7b63-41fc-a5be-c0f0e09ca283)

**Comment by EngrPips** [comment]
Hello @Aishwaria8786 , maybe you should send the transaction hash of the transaction you sent to solve the challenge. So that we can have insight to what the transaction is.

**Reply by Aishwaria8786** [reply, marked as answer]
I was able to import it after 7-10 minutes. I think I put a whitespace somewhere by mistake.

**Comment by EngrPips** [comment]
Glad it works now
